src/pages/TestDrive/TestDrive.tsx

```diff
@@ -0,0 +1,117 @@
+import { Button, Container, Form, Card } from "react-bootstrap";
+import { useLocation, useParams, Link, useNavigate } from "react-router-dom";
+import React, { useState } from "react";
+import "./TestDrive.scss";
+function TestDrive() {
+  let navigate = useNavigate();
+  const [mobileValid, setMobileValid] = useState(false);
+  const [emailValid,setEmailValid] = useState(false);
```

Comment:
do we need this we already have pattern guard in respective form inputs

Reply:
Yes removed these 2 cases